### PR TITLE
updated ReadMe

### PR DESCRIPTION
**Pull request by Pragya82001**

### CODE REVIEW

File: Data Structures/Linked Lists/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+● The last node has a reference to null.
+
+
+
+
+II) Linked list class: 
+
+
+class Node {
```
Comment (msboffl) [inline Data Structures/Linked Lists/README.md:26]: Can you please add the code in these backticks ```code ```

refer the below link
[github link](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks)

File: Data Structures/Linked Lists/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+public :
```
Comment (msboffl) [inline Data Structures/Linked Lists/README.md:28]: same as above comment

**Review by msboffl — CHANGES_REQUESTED** [review]

File: Data Structures/Linked Lists/README.md (hunk 2 above)
Comment (Pragya82001) [reply Data Structures/Linked Lists/README.md:28]: You are not doing this under hacktoberfest tag ?

Comment (msboffl) [reply Data Structures/Linked Lists/README.md:28]: I forgot to add, I will add tomorrow

Comment (Pragya82001) [reply Data Structures/Linked Lists/README.md:28]: okay

**Comment by msboffl** [comment]
Approved

**Pull request merged**